app/script.js

```diff
@@ -1,13 +1,71 @@
 import Aragon from '@aragon/client'
+import { of } from 'rxjs/observable/of'
 
 const app = new Aragon()
 
+const INITIALIZATION_TRIGGER = Symbol('INITIALIZATION_TRIGGER')
+
 const initialState = {
   nodes: []
 }
 
-app.store(async (state, event) => {
-  console.log('hiiiiiiii')
-  if (state === null) state = initialState
+app.store(async (state, { event, returnValues }) => {
+  switch (event) {
+    case INITIALIZATION_TRIGGER:
```

Comment:
Can this code be reached? I think it needs and event in the smart contract for when the initialize function gets called.

I'm ok with leaving it here for now as a reminder.